chef-config/lib/chef-config/windows.rb

```diff
@@ -17,8 +17,9 @@
 
 module ChefConfig
   def self.windows?
-    if @node
-      platform = @node[:platform] || @node[:platform_family] || @node[:os]
+    @node ||= nil
+    if @node && @node[:os]
```

Comment:
Wouldn't this always result in returning `true` if @node is present and has an :os key, even on non-windows systems?